po/LINGUAS

```diff
@@ -1,6 +1,7 @@
 be
 be@latin
 bg
+cs_CZ
```

Comment:
cs_CZ is a known exception. cs in the past meant Czechoslovakia. And then we have the czech (CZ) part.